RecoVertex/AdaptiveVertexFinder/interface/SVTimeHelpers.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __RecoVertex_AdaptiveVertexFinder_SVTimeHelpers_h__
+#define __RecoVertex_AdaptiveVertexFinder_SVTimeHelpers_h__ 
+
+#include "RecoVertex/VertexPrimitives/interface/TransientVertex.h"
+#include "TrackingTools/TransientTrack/interface/TransientTrack.h"
+
+#include "FWCore/Utilities/interface/isFinite.h"
+
+namespace svhelper{
+
+  inline void updateVertexTime(TransientVertex& vtx) {
+    const auto& trks = vtx.originalTracks();
+    double meantime = 0., expv_x2 = 0., normw = 0., timecov = 0.;		  
+    for( const auto& trk : trks ) {
+      if( edm::isFinite(trk.timeExt()) ) {
+	const double time = trk.timeExt();
+	const double inverr = 1.0/trk.dtErrorExt();
```

Comment:
Are we sure that `trk.dtErrorExt()` will never be zero?

Reply:
Yes, if the time is a valid number its error cannot be zero.